src/NzbDrone.Core/Download/Clients/Tribler/TriblerDownloadClientProxy.cs

```diff
@@ -0,0 +1,137 @@
+                .Accept(HttpAccept.Json);
+
+            requestBuilder.Headers.Add("X-Api-Key", settings.ApiKey);
+
+            requestBuilder.LogResponseContent = true;
+
+            return requestBuilder;
+        }
+
+        public string GetBaseUrl(TriblerDownloadSettings settings, string relativePath = null)
+        {
+            var baseUrl = HttpRequestBuilder.BuildBaseUrl(settings.UseSsl, settings.Host, settings.Port, settings.UrlBase);
+            baseUrl = HttpUri.CombinePath(baseUrl, relativePath);
+
+            return baseUrl;
+        }
+
+        private T ProcessRequest<T>(HttpRequestBuilder requestBuilder)
+            where T : new()
+        {
+            return ProcessRequest<T>(requestBuilder.Build());
+        }
+
+        private T ProcessRequest<T>(HttpRequest requestBuilder)
+            where T : new()
+        {
+            var httpRequest = requestBuilder;
+
+            HttpResponse response;
+
+            _logger.Debug("Url: {0}", httpRequest.Url);
+
+            try
+            {
+                response = _httpClient.Execute(httpRequest);
+            }
+            catch (HttpException ex)
+            {
+                if (ex.Response.StatusCode == System.Net.HttpStatusCode.Unauthorized)
+                {
+                    throw new DownloadClientAuthenticationException("Unauthorized - AuthToken is invalid", ex);
+                }
+
+                throw new DownloadClientUnavailableException("Unable to connect to Tribler. Status Code: {0}", ex.Response.StatusCode, ex);
+            }
+
+            return Json.Deserialize<T>(response.Content);
+        }
+
+        public GetTriblerSettingsResponse GetConfig(TriblerDownloadSettings settings)
+        {
+            var configRequest = getRequestBuilder(settings, "settings");
+            return ProcessRequest<GetTriblerSettingsResponse>(configRequest);
+        }
+
+        public ICollection<File> GetDownloadFiles(TriblerDownloadSettings settings, Download downloadItem)
+        {
+            var filesRequest = getRequestBuilder(settings, "downloads/" + downloadItem.Infohash + "/files");
+            return ProcessRequest<GetFilesResponse>(filesRequest).Files;
+        }
+
+        public ICollection<Download> GetDownloads(TriblerDownloadSettings settings)
+        {
+            var downloadRequest = getRequestBuilder(settings, "downloads");
+            var downloads = ProcessRequest<DownloadsResponse>(downloadRequest);
+            return downloads.Downloads;
+        }
+
+        public void RemoveDownload(TriblerDownloadSettings settings, DownloadClientItem item, bool deleteData)
+        {
+            var deleteDownloadRequestObject = new RemoveDownloadRequest
+            {
+                RemoveData = deleteData
+            };
+
+            var deleteRequestBuilder = getRequestBuilder(settings, "downloads/" + item.DownloadId.ToLower());
+            deleteRequestBuilder.Method = System.Net.Http.HttpMethod.Delete;
```

Comment:
import the namespace (`System.Net.Http`) so you can just use `HttpMethod.Delete`